### DISCUSSION TITLE
How about shadcn/ui?

### DISCUSSION THREAD

**Discussion by internalG** [scosman/CMSaasStarter#81] [Ideas]
Your choices almost match everything I researched except DaisyUI. And https://www.shadcn-svelte.com/ looks very cool. So why not?

**Comment by scosman** [comment]
Moved to discussion. This isn't an issue as much as an idea.

Per prior conversations: several folks have said they want this and may create a [community extension](https://github.com/CriticalMoments/CMSaasStarter?tab=readme-ov-file#extensions) for it. A PR is welcome!

**Reply by A7iDev** [reply]
https://github.com/kizivat/saas-kit

**Reply by scosman** [reply]
Cool! Want to link to it in the community extensions part of the readme?

**Comment by kryanbeane** [comment]
I do have future plans to do this once my side project is released and I have more time. 

I will definitely get part of it going and then push it to a community extension for people to help with!

edit: I have no idea if/when I could get to doing it though, so if someone else was to start it that'd be amazing and I'd 100% contribute as much as i can

**Comment by ghost** [comment]
Shadcn-svelte is nice (great) but it's run by one person (Huntabyte). He's a powerhouse of a dev but what if he's pulled elsewhere?

**Comment by scosman** [comment]
Curious: what do people like about shadcn? I love the mono look of it, but you could accomplish the same with a good DaisyUI theme (could be a good addition here). 

Regarding coding: the react-like components for rendering replacing HTML+CSS seem to go against the "svelte way" of doing things and seem out of place in a svelte boilerplate. It really looks like a react-tool shoehorned into svelte world.

With DaisyUI the tailwind compiler works its magic to compile out any unused css. The shadcn import style will reduce unused code, but not fully minimize css.

Let me know what I'm missing!

**Comment by scosman** [comment]
In case people miss it, @kizivat has a fork using shadcn/ui: https://github.com/kizivat/saas-kit

I think fork makes the most sense. If anyone else makes one, post it here for others to find.
